examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3658,7 +3658,7 @@ class GLTFParser {
 					// Node may be a Group (glTF mesh with several primitives) or a Mesh.
 					node.traverse( function ( object ) {
 
-						if ( object.isMesh === true && object.morphTargetInfluences ) {
+						if ( object.morphTargetInfluences ) {
```

Comment:
It seems the above comment `// Node may be a THREE.Group (glTF mesh with several primitives) or a THREE.Mesh.` is now not correct anymore.

Besides, do you mind sharing a glTF for testing that is currently not correctly loaded (but fixed with this PR)?

Reply:
Done, thanks. Also, I attach a glTF file that shows the error.
[slab deformation.gltf.zip](https://github.com/mrdoob/three.js/files/7466782/slab.deformation.gltf.zip)
.